test/integration_tests/python/test_jobs.py

```diff
@@ -94,3 +94,5 @@ def test_jobs(with_hierarchy, with_gear, with_invalid_gear, as_user, as_admin):
 
     r = as_user.post('/jobs/' + job1_id + '/retry')
     assert r.ok
+
+    db.jobs.remove()
```

Comment:
That sounds like a great idea. Let's leave this code as-is for this PR, it sounds like we have a good path forward. I don't think running stuff in parallel is hugely important, but I gotta say doing it for the SDK tests feels very nice :slightly_smiling_face: